src/circleci/rollcage/core.clj

```diff
@@ -1,6 +1,7 @@
 (ns circleci.rollcage.core
   (:require
     [clojure.string :as string]
+    [clojure.tools.logging :refer (infof errorf)]
```

Comment:
Let's not bring this extra dependency into Rollcage.

I would rather do this by supplying an optional `failure-fn` to the client that will be called with failures.

```Clojure
(defn send-item [endpoint item]
  (try
    (let [result (post endpoint {:body (json/generate-string item {:key-fn snake-case})
                                 :content-type :json})]
      (json/parse-string (:body result) true))
    (catch Exception e
      (when-let [failure-fn (:failure-fn client)]
        (failure-fn item e))